webpack.config.js

```diff
@@ -5,10 +5,19 @@ const ZipPlugin = require('zip-webpack-plugin');
 const FaviconsPlugin = require('favicons-webpack-plugin')
 const CopyPlugin = require('copy-webpack-plugin')
 const HtmlPlugin = require('html-webpack-plugin')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
```

Comment:
Why don't using a style that uses a few lines above? 🤔 
I think `CleanPlugin` is more good name.